jazkarta/shop/browser/controlpanel.py

```diff
@@ -151,13 +149,13 @@ def endDateString(self):
         else:
             if self.orders_exist:
                 ed = self.to_datetime(self.most_recent_order_date,
-                    '%Y-%m-%d %I:%M %p')
+                    '%Y-%m-%d %I:%M %p').date()
             else:
-                ed = datetime.datetime.today()
-        return ed.strftime(u'%Y-%m-%d')
+                ed = datetime.datetime.today().date()
```

Comment:
Not a problem, but you could use `datetime.date.today()`

Reply:
:+1: